01_variables/swap_the_values_of_two_variable.py

```diff
@@ -1,2 +1,15 @@
 # Write a program to swap the values of two variables 
-# for more info on this quiz, go to this url: http://www.programmr.com/swap-values-two-variable-1
\ No newline at end of file
+# for more info on this quiz, go to this url: http://www.programmr.com/swap-values-two-variable-1
+
+
+def swap_variables():
+    x = input("Enter value of X: ")
+    y = input("Enter value of Y: ")
+    a = x
```

Comment:
There is a better trick to swap values that you will learn after going through the list and tuple sections

```python
x = 5
y = 7
x, y = y, x  # x = 7, y = 5
```
This works because of multiple assignment feature in tuples and recently lists
You don't have to worry about this for now, you will learn it in tuples and list section